app/api/events.py

```diff
@@ -468,9 +468,11 @@ def before_update_object(self, event, data, view_kwargs):
         :param view_kwargs:
         :return:
         """
-
         if has_access('is_admin') and data.get('deleted_at') != event.deleted_at:
-            event.deleted_at = data.get('deleted_at')
+            if len(event.orders) != 0:
```

Comment:
Check for only active orders please

Reply:
@iamareebjamal We concluded these possibilities of scams -

 - User can create an event and people buy it  and user do not show up on the venue and deletes the event from eventyay just after it finishes, as event will become past event.

If we change the condition and only restrict events with active orders to be deleted, how can we avoid above mentioned scams?

Reply:
But he will not be able to delete event if it has active orders? In what cases an order can be deleted by a user?

Reply:
An order can never be deleted by a user. We are not giving them this feature as of now